src/algebra/group/to_additive.lean

```diff
@@ -230,7 +256,30 @@ In the `mul_comm'` example above, `to_additive` maps:
 * `x * y` to `x + y` and `y * x` to `y + x`, and
 * `comm_semigroup.mul_comm'` to `add_comm_semigroup.add_comm'`.
 
-Even when `to_additive` is unable to automatically generate the additive
+### Heuristics
+
+`to_additive` uses heuristics to determine whether a particular identifier has to be
+mapped to its additive version. The basic heuristic is
+
+* Only map an identifier to its additive version if its first argument doesn't
+  contain any unapplied identifiers.
```

Comment:
What exactly does this mean? Can you provide an example of something that is mapped and something that isn't, and which part of them is the "unapplied identifier"?

Reply:
I meant the next couple of lines as examples, but I'll expand it